tools/tools.go

```diff
@@ -1,9 +1,19 @@
-//go:build tools
-// +build tools
+//go:build generate
 
 package tools
 
 import (
+	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
+	_ "github.com/hashicorp/copywrite"
 	_ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"
-	_ "golang.org/x/tools/cmd/goimports"
+	_ "mvdan.cc/gofumpt"
 )
+
+// Generate copyright headers
+//go:generate go run github.com/hashicorp/copywrite headers -d .. --config ../.copywrite.hcl
+// Format Terraform code for use in documentation.
+// If you do not have Terraform installed, you can remove the formatting command, but it is suggested
+// to ensure the documentation is formatted properly.
+//go:generate terraform fmt -recursive ../examples/
+// Generate documentation.
+//go:generate go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs generate --provider-dir ..
```

Comment:
I don't like keeping it both in makefile and here, these two rules for doc generation differ slightly, we should stick with one. Because we also would like to perform check on it, I would prefer keeping generation and check inside makefile

Reply:
they are the same command. `--provider-dir ..` just indicates that the provider directory is one level higher, since it is running in the tools module.

We can keep it as it is now, but again this is convention when you look at all the other TF providers.

Reply:
I know that they are the same; what I meant is that if they are the same, they should be just one command and not two. Closing for now.